app/src/test/kotlin/io/intrepid/skotlinton/testutils/LiveDataTestBase.kt

```diff
@@ -0,0 +1,10 @@
+package io.intrepid.skotlinton.testutils
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import org.junit.Rule
+
+abstract class LiveDataTestBase {
```

Comment:
This was just used for `LiveDataObserverTest`. I'm fine with just removing this and add the rule to `ViewModelTestBase` and `LiveDataObserverTest` since it probably won't be used elsewhere